wget_retry.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+RETRIES=10
+SLEEPTIME=5
+COUNT=0
+while [ $COUNT -lt $RETRIES ]
+do
+    wget --progress=dot:giga --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 --tries 20 --continue "$@"
```

Comment:
@mattface as far as I know wget, it won't retry if it gets a 404, but just fail.

I looked at the various config options, and this should make the retry work also with 404: `--retry-on-http-error=404`

Obviously, assuming you want to retry with 404 as well.